Day6-Assignment/src/Components/SearchBar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {useState} from 'react'
+
+/**
+	 * @description: function is called on Submit of the form, This function describes the functionality of Search
+	 * @param1: searchText : entered search text
+     * @param2: user:props passed to the function to provide user data
+*/
+
+const searchMemeber = (searchText,user) =>{
+ return user.filter((item)=>{ return item.name.toLowerCase().includes(searchText.toLowerCase())})
```

Comment:
should use option chaining as sometimes name property is not provided.

Reply:
ok will incorporate this from next Assignment